tests/installation/opensuse_welcome.pm

```diff
@@ -16,10 +16,11 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use x11utils 'untick_welcome_on_next_startup';
+use x11utils qw(untick_welcome_on_next_startup workaround_boo1170586);
 
 sub run {
-    assert_screen("opensuse-welcome");
+    assert_screen([qw(opensuse-welcome opensuse-welcome-boo1170586)]);
+    workaround_boo1170586 if match_has_tag("opensuse-welcome-boo1170586");
```

Comment:
I think it can be clearer if you move the `assert_screen` call into a function as well which you can call `handle_opensuse_welcome`. This also avoids the not so clear function name `workaround_boo1170586`.

Reply:
So, you would also move the untick in this `handle_opensuse_welcome`?